setup.py

```diff
@@ -185,6 +188,9 @@ def create_extension(ExtType, name, sources, dependencies, extra_link_args, extr
         extra_compile_args.append('-stdlib=libc++')
     if os.getenv('CONDA_PREFIX'):
         include_dirs.append(os.path.join(os.getenv('CONDA_PREFIX'), "include"))
+        include_dirs.append(os.path.join(os.getenv('CONDA_PREFIX'), "Library", "Include"))
```

Comment:
Ditto here.

Reply:
Sorry, what do you mean "not portable" please? This is similar with the line above, but adding one more include dir which contains protobuf header files.